include/dsn/tool-api/task_tracker.h

```diff
@@ -166,11 +167,16 @@ class task_tracker
     // return not finished task count
     int cancel_but_not_wait_outstanding_tasks();
 
+    void set_failure() { _all_task_success = false; }
+
+    bool all_tasks_success() const { return _all_task_success; }
+
 private:
     friend class trackable_task;
     const int _task_bucket_count;
     ::dsn::utils::ex_lock_nr_spin *_outstanding_tasks_lock;
     dlink *_outstanding_tasks;
+    bool _all_task_success{true};
```

Comment:
It seems the default value of `_all_task_success` should be false, it will be true through function `set_succeed`.

Reply:
Oh, it's mistake..

Reply:
done